distribution/src/main/release/samples/corba/bank/pom.xml

```diff
@@ -59,6 +59,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
@jimma @asoldano I have updated all samples which use Spring 6 / Spring Boot 3 to require JDK-17, otherwise they are not runnable